src/youtube-player/youtube-player.ts

```diff
@@ -244,14 +244,15 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
 
     // Set up side effects to bind inputs to the player.
     playerObs.subscribe(player => {
+      if (!player) {
```

Comment:
This means that the old player won't be cleared out if a new `undefined` one comes in. We can do something like this instead:

```
this._player = player;
this._playerChanges.next(player);

if (player) {
  if (this._pendingPlayerState) {
    this._initializePlayer(player, this._pendingPlayerState);
  }
  
  this._pendingPlayerState = undefined;
}
```